students/BrianB/lesson04/dic_lab.py

```diff
@@ -0,0 +1,28 @@
+#! bin/usr/env python3
```

Comment:
this should be:

#!/usr/bin/env python3

but since you clearly aren't using it -- you can just leave it off.